code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -322,8 +322,8 @@
 
 //Laser carbine//
 /obj/item/gun/projectile/automatic/lasercarbine
-	name = "\improper IK-60 laser carbine"
-	desc = "A compact, twin barrelled carbine that uses disposable laser cartridges rather than an internal power cell. Utilized by the Nanotrasen Navy for combat operations."
+	name = "\improper IK-M2 laser carbine"
+	desc = "A compact, Warp-Tec Industries twin barrelled carbine that uses disposable laser cartridges rather than an internal power cell. Utilized by the Nanotrasen response teams for combat operations."
```

Comment:
```suggestion
	desc = "A compact, Warp-Tac Industries twin barrelled carbine that uses disposable laser cartridges rather than an internal power cell. Utilized by the Nanotrasen response teams for combat operations."
```